app/ante/eth.go

```diff
@@ -164,16 +163,12 @@ func (egcd EthGasConsumeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 
 		evmDenom := egcd.evmKeeper.GetEVMDenom(ctx)
 
-		fees, err := egcd.evmKeeper.DeductTxCostsFromUserBalance(
-			ctx,
-			*msgEthTx,
-			txData,
-			evmDenom,
-			baseFee,
-			homestead,
-			istanbul,
-			london,
-		)
+		fees, err := evmkeeper.VerifyFee(ctx, txData, evmDenom, baseFee, homestead, istanbul)
```

Comment:
Seems easy to confuse `evmkeeper` the import with `evmKeeper` the Keeper. Could we rename the import to `keeper`?